tide_publication.install

```diff
@@ -1005,3 +1005,23 @@ function tide_publication_update_8013() {
     }
   }
 }
+
+/**
+ * Removed featured_news option from the landing_page widget.
+ */
+function tide_publication_update_8014() {
+  $field = FieldConfig::loadByName('node', 'publication_page', 'field_landing_page_component');
+  if ($field) {
+    $handler_settings = $field->getSetting('handler_settings');
+    if (isset($handler_settings['target_bundles'])) {
+      if (isset($handler_settings['target_bundles']['featured_news'])) {
```

Comment:
You could also remove the elements using array_diff without having to check for them

```
// Remove the elements who's values are yellow or red
$arr = array_diff($arr, ["yellow", "red"]);
```